app/src/main/java/org/openobservatory/ooniprobe/domain/MeasurementsManager.java

```diff
@@ -113,8 +114,8 @@ public void downloadReport(Measurement measurement, DomainCallback<String> callb
         //measurement.getUrlString will return null when the measurement is not a web_connectivity
         apiClient.getMeasurement(measurement.report_id, measurement.getUrlString()).enqueue(new GetMeasurementsCallback() {
             @Override
-            public void onSuccess(ApiMeasurement.Result result) {
-                downloadMeasurement(result, callback);
```

Comment:
Can we get rid of the `downloadMeasurement` method now that we download the measurement directly and don't have to proxy it through the `/measurements` query?